packages/models-library/src/models_library/api_schemas_storage.py

```diff
@@ -149,7 +153,7 @@ class FileMetaDataGet(BaseModel):
     )
 
     model_config = ConfigDict(
-        extra="forbid",
+        extra="ignore",
```

Comment:
Intentional?

Reply:
he is ignoring it ;-b

Reply:
@giancarloromeo  yes. that is what I wrote to you in the chat yesterday.